integr8ly/integreatly-operator
# Mgdapi 4402

# Issue link
[MGDAPI-4402](https://issues.redhat.com/browse/MGDAPI-4402)

# What
- Changed 3scale imports to reflect the new path being used in 3scale-operator
- Bumped 3scale-operator version from 0.7.0 to 0.9.0 (needed for external components)
- Replaced high availability with externalComponents in the three scale reconciler to prevent the API manager being reconciled constantly

# Verification steps
Install RHOAM on CCS cluster and verify that 3scale installs successfully and that the ApiManager CR doesnt update constantly ie `metadata.generation < 10 ` after installation.

1. Check out this branch 
2. log in to a CCS cluster
3. Run the following commands
```
INSTALLATION_TYPE=managed-api  make cluster/prepare/local
```
```
INSTALLATION_TYPE=managed-api USE_CLUSTER_STORAGE=false make deploy/integreatly-rhmi-cr.yml 
```
```
INSTALLATION_TYPE=managed-api  make code/run
```
5. After RHOAM installs successfully check the APIManager CR  in the redhat-rhoam-3scale namespace and make sure that the generation in the metadata is less than 10
